src/components/inputs/money-input/money-input.js

```diff
@@ -328,59 +370,55 @@ export default class MoneyInput extends React.Component {
     }
   };
 
-  containerRef = React.createRef();
   amountInputRef = React.createRef();
 
   render() {
+    const hasNoCurrencies = this.props.currencies.length === 0;
+    const hasWarning = this.props.hasCurrencyWarning;
+    const hasError = this.props.hasCurrencyError;
+    const currencySelectStyles = createCurrencySelectStyles({
+      hasWarning,
+      hasError,
+      hasNoCurrencies,
+    });
+    const options = this.props.currencies.map(currencyCode => ({
+      label: currencyCode,
+      value: currencyCode,
+    }));
+
+    const option = do {
+      const matchedOption = options.find(
+        o => o.value === this.props.value.currencyCode
```

Comment:
Nitpick: I thought we do not use one letter abbreviations :)? I don't mind too much, at some level one just runs out of names. I remember using `optionCandidate` the other day which I still don't know if it's better.

Reply:
Yeah, I ran out of names :( I like `optionCandidate`!